packages/engine/src/scene/classes/PostProcessing.ts

```diff
@@ -0,0 +1,149 @@
+import { Object3D } from "three";
+import { BlendFunction } from "../../renderer/postprocessing/blending/BlendFunction";
+import { BloomEffect } from "../../renderer/postprocessing/BloomEffect";
+import { BrightnessContrastEffect } from "../../renderer/postprocessing/BrightnessContrastEffect";
+import { ColorDepthEffect } from "../../renderer/postprocessing/ColorDepthEffect";
+import { DepthOfFieldEffect } from "../../renderer/postprocessing/DepthOfFieldEffect";
+import { FXAAEffect } from "../../renderer/postprocessing/FXAAEffect";
+import { HueSaturationEffect } from "../../renderer/postprocessing/HueSaturationEffect";
+import { LinearTosRGBEffect } from "../../renderer/postprocessing/LinearTosRGBEffect";
+import { OutlineEffect } from "../../renderer/postprocessing/OutlineEffect";
+import { SSAOEffect } from "../../renderer/postprocessing/SSAOEffect";
+import { ToneMappingEffect } from "../../renderer/postprocessing/ToneMappingEffect";
+
+/**
+ * @author Abhishek Pathak <abhi.pathak401@gmail.com>
+ */
+export default class PostProcessing extends Object3D{
```

Comment:
This should be a component that extends Component from engine/ecs.
However, we haven't unified our game engine and editor (the editor is Mozilla Spoke).
The editor doesn't use the engine, so there is no way to attach a component.
For now, let's keep this class, but we will basically need to duplicate it in order to make a component for the engine side. Later, when we unify the editor and engine, we can delete this class and only use entities + components.